packages/web-components/src/components/va-button-icon/va-button-icon.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  Component,
+  Event,
+  EventEmitter,
+  Host,
+  h,
+  Listen,
+  Prop,
+  Element,
+} from '@stencil/core';
+import classnames from 'classnames';
+
+/**
+ * @componentName Button - Icon
+ * @nativeHandler onClick
+ * @maturityCategory caution
+ * @maturityLevel candidate
+ */
+
+@Component({
+  tag: 'va-button-icon',
+  styleUrl: 'va-button-icon.scss',
+  shadow: true,
+})
+export class VaButtonIcon {
+  @Element() el: HTMLElement;
+
+  /**
+   * If `true`, the component-library-analytics event is disabled.
+   */
+  @Prop() disableAnalytics?: boolean = false;
+
+  /**
+   * If `true`, the click event will not fire.
+   */
+  @Prop({ reflect: true }) disabled?: boolean = false;
+
+  /**
+   * The aria-label of the component.
+   */
+  @Prop() label?: string; // could use this.el.getAttribute('aria-label') but this is more explicit
+
+  /**
+   * The type of button this will render as, currently a limited number of options
+   */
+
+  private buttonTypeMap = {
+    // eslint-disable-next-line i18next/no-literal-string
+    'change-file': { icon: 'attach_file', text: 'Change File' },
+    // eslint-disable-next-line i18next/no-literal-string
+    'delete': { icon: 'delete', text: 'Delete' },
+    // eslint-disable-next-line i18next/no-literal-string
+    'cancel': { icon: 'close', text: 'Cancel' },
+  };
+  @Prop() buttonType: keyof typeof this.buttonTypeMap;
+
+  /**
+   * The event used to track usage of the component.
+   */
+  @Event({
+    bubbles: true,
+    composed: true,
+    eventName: 'component-library-analytics',
+  })
+  componentLibraryAnalytics: EventEmitter;
+
+  private handleClick = (): void => {
+    if (!this.disableAnalytics) {
+      const detail = {
+        componentName: 'va-button',
```

Comment:
This should probably be `va-button-icon`

Reply:
Confirmed with Matt Dingee that he's OK with keeping it a 'va-button' component for the analytics event, with the caveat of making it a "tertiary" type instead of "primary".

Reply:
@Andrew565 That's not my reading of the [Slack convo](https://dsva.slack.com/archives/G01BJ3ESXL4/p1713877193656529):

> I'd really like to be able to know if users are interacting with this new button style. This is one where we want to be sure that it has the affordance of a button and users are not missing it.

How are we supposed to know they are interacting with the new button style if we don't add the name of the new button component?

Reply:
The "label" property of the analytics event will say either "Change File", "Delete", or "Cancel".